### PR TITLE
[Storage] revert other changes in storage

### PR DESCRIPTION
**Pull request by blueww**
And add change log.
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage/Blob/Cmdlet/GetAzureStorageContainer.cs [hunk 1]
```diff
@@ -29,7 +29,6 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// List azure storage container
     /// </summary>
     [Cmdlet("Get", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainer", DefaultParameterSetName = NameParameterSet),OutputType(typeof(AzureStorageContainer))]
-    [Alias("Get-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainerAcl", "Get-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "DatalakeGen2FileSystem")]
```
Comment (msJinLei) [inline src/Storage/Storage/Blob/Cmdlet/GetAzureStorageContainer.cs]: @blueww confirmed, not alias for previous sprint.

File: src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainer.cs [hunk 2]
```diff
@@ -27,7 +27,6 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// create a new azure container
     /// </summary>
     [Cmdlet("New", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainer"),OutputType(typeof(AzureStorageContainer))]
-    [Alias("New-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "DatalakeGen2FileSystem")]
```
Comment (msJinLei) [inline src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainer.cs]: @blueww confirmed, not alias for previous sprint.

File: src/Storage/Storage/Blob/Cmdlet/RemoveAzureStorageContainer.cs [hunk 3]
```diff
@@ -27,7 +27,6 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// remove specified azure container
     /// </summary>
     [Cmdlet("Remove", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainer", SupportsShouldProcess = true),OutputType(typeof(Boolean))]
-    [Alias("Remove-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "DatalakeGen2FileSystem")]
```
Comment (msJinLei) [inline src/Storage/Storage/Blob/Cmdlet/RemoveAzureStorageContainer.cs]: @blueww confirmed, not alias for previous sprint.

File: src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainer.cs (hunk 2 above)
Comment (blueww) [reply src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainer.cs]: Confirm: This alias only on preview module Az.Storage before.

File: src/Storage/Storage/Blob/Cmdlet/RemoveAzureStorageContainer.cs (hunk 3 above)
Comment (blueww) [reply src/Storage/Storage/Blob/Cmdlet/RemoveAzureStorageContainer.cs]: Confirm: This alias only on preview module Az.Storage before.

File: src/Storage/Storage/Blob/Cmdlet/GetAzureStorageContainer.cs (hunk 1 above)
Comment (msJinLei) [reply src/Storage/Storage/Blob/Cmdlet/GetAzureStorageContainer.cs]: breaking changes. should not remove the alias

**Pull request merged**